components/ILIAS/IndividualAssessment/PRIVACY.md

```diff
@@ -0,0 +1,116 @@
+- **File**: Examiners can add a file to the record. This field can be set as required
+  in the Settings.
+- **Changes after finalization:** Users with the corresponding permission can change
+  a record after it was finalized. Such a change is logged and stores the User ID of
+  the Changer, as well as time and date of the change.
+- **Contact Information**: In the tab `Settings > Info Settings`, contact information
+  can be entered. This may include a person's name, responsibility, phone, email and
+  consultation hours.
+
+## Data being presented
+
+- **Users with high-level permissions can see:**
+    - **Users**: User search results (last and first name, login name of a user) can
+      be seen in the toolbar of the overview and while adding members.
+    - **Participants**: The name of Participants is presented in the overview screen
+      and while adding or editing a Participant Record.
+    - **Examiner**: The name of the Examiner is shown in overviews and editing screens.
+    - **Location, time and date of an assessment** in overview and during editing.
+    - **Changed after finalization**: If the record was edited after finalization, the
+      name of the Changer, as well as date and time of this change will be shown.
+    - **Grading**: is shown in the overview and the editing view of the Participant Record
+    - **Record Notes**: Both the public and the internal record note are shown in the
+      overview and editing view of  records.
+    - **File**: Attached files can be downloaded from the overview screen.
+    - **Contact Information** can be set and viewed within the Settings.
+- **General users** can only access the top level info page and see a few items:
+    - their **Grading**
+    - the public **Record Note** of their record
+    - the **File** uploaded to their record if the visibility option was set
+    - the manually set **Contact Information** from the general settings
+
+## Data being deleted
+
+- When deleting a single Participant record before finalizing it, the following personal
+  data stored so far will be deleted:
+    - reference to user ID for Participant
+    - Location, time and date of an assessment
+    - Grading
+    - both Record Notes
+    - uploaded file
+- After finalizing, Participant Records cannot be deleted individually. The entire
+  Individual Assessment object needs to be deleted to remove data.
+- When deleting the entire Individual Assessment, all records will be deleted and the
+  following personal data potentially stored with it:
+    - user ID for Participant, Examiner, Changer
+    - time and date of the last change
+    - Location, time and date of an assessment
+    - Grading
+    - both Record Notes
+    - uploaded file
+    - manually provided, optional contact information
+
+## Data being exported
+
+- Only the settings of the Individual Assessment and no Participant Records are exported.
+  Therefore the only sensitive data included at this point is:
```

Comment:
You can slightly change to:
Therefore, the only sensitive data included at this point is manually provided, optional contact information.